### ISSUE TITLE
MeshCommander unable to connect to AMT device

### ISSUE THREAD

**Issue by ZENAdmin-Ops** [Ylianst/MeshCommander#86]
Hello,

This is a question, rather than an issue.

I have configured MEBx via manual setup and enabled AMT on a Dell Precision 3660.

So, I know the user and the password.

I can connect to the WebUI via https://ip_address:16993

MeshCommander can detect the device via scan, but when I try to connect, the connection times out.

I note in your MeshCommander tutorial video, that when configuring TLS, the AMT device starts out in non-TLS mode before configuring the certificate and enabling TLS.

In trying to troubleshoot this system, I downloaded some Intel tool onto the PC that said that AMT was already in TLS-mode (as shipped). I'm not sure if this might be the reason that MeshCommander is unable to connect.

Basically I'm after troubleshooting suggestions.

**Comment by jsastriawan** [comment]
Please edit your device connection to use Digest TLS instead of Digest None. Then MeshCommander should connect to your device using TLS at pprt 16993.
